frame/crowdloan-rewards/src/lib.rs

```diff
@@ -230,7 +240,7 @@ pub mod pallet {
 		/// ```haskell
 		/// proof = sign (concat prefix (hex reward_account))
 		/// ```
-		#[pallet::weight(<T as Config>::WeightInfo::associate(TotalContributors::<T>::get()))]
+		#[pallet::weight((<T as Config>::WeightInfo::associate(TotalContributors::<T>::get()), Pays::No))]
```

Comment:
Hm, are both Pays::No needed? is this Pays::No males errors also free?

Reply:
Not sure actually, I will run some tests and check

Reply:
Removing the Pays::No in the weight annotation makes the valid transactions paid (valid_associate_transactions_are_free fails)

Reply:
I'm unsure but I think that keeping the weight allow to avoid a block being full